services/frontend-service/src/ui/components/ServiceLane/ServiceLane.scss

```diff
@@ -1,34 +1,40 @@
-.service-lane__header{
-  background: var(--mdc-theme-primary);
-  color: var(--mdc-theme-on-primary);
-  display: flex;
-  justify-content: space-between;
-  padding: 13px 20px;
-  border-radius: 10px;
-  height: 62px;
+.service-lane__header {
+	background: var(--mdc-theme-primary);
```

Comment:
Restore the spacing that was used before

Reply:
done!